openrpc/src/anchor-platform/schemas/platform_transaction.json

```diff
@@ -0,0 +1,105 @@
+{
+  "platform_transaction": {
+    "title": "Platform Transaction",
+    "description": "A platform transaction object",
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "#/components/schemas/transaction_id"
+      },
+      "sep": {
+        "$ref": "#/components/schemas/sep"
+      },
+      "kind": {
+        "$ref": "#/components/schemas/kind"
+      },
+      "status": {
+        "$ref": "#/components/schemas/status"
+      },
+      "amount_expected": {
```

Comment:
`type` is missing

Reply:
They are all included in the reference object?